explorer/client/src/api.ts

```diff
@@ -1,13 +1,14 @@
 import 'whatwg-fetch'
 import { Query } from './reducers/search'
+import { ADMIN_USERNAME_HEADER, ADMIN_PASSWORD_HEADER } from './constants'
 
 const base = () => `${document.location.protocol}//${document.location.host}`
 
-const getJobRuns = async (
+export async function getJobRuns(
```

Comment:
Just for future reference, maybe we should look into spiltting out the `api` module in `operator_ui` and reusing it across our projects. A lot of code duplication here